libraries/aws_cloudformation_stacks.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsCloudformationStacks < AwsResourceBase
+  name 'aws_cloudformation_stacks'
+  desc 'Verifies settings for aws CloudFormation Stacks'
+
+  example "
+    describe aws_cloudformation_stacks do
+      it { should exist }
+    end
+  "
+
+  attr_reader :table
+
+  FilterTable.create
+             .register_column(:names, field: :name)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    cloudformation_stacks_rows = []
+    catch_aws_errors do
+      @api_response = @aws.cloudformation_client.describe_stacks
+    end
+    return [] if !@api_response || @api_response.empty?
+    @api_response.stacks.each do |cf_stack|
+      cloudformation_stacks_rows+=[{ name: cf_stack.stack_name }]
```

Comment:
why is it restricted to tags only? I guess we should add other attributes through which the cloudformation stacks could be filtered

Reply:
For what my team needs and In practice i would think the stack name is a good tag for filtering because any further filtering can be done in-depth iteratively.

Reply:
I was talking about this [filters](https://docs.chef.io/inspec/glossary/#filter-criteria).. I think we could add all boolean, datetime & identifiers keys to support this..

Reply:
edited that now